README.md

```diff
@@ -149,4 +149,88 @@ As reviewers, a PR should be only approved if following conditions are met (In o
 - git add \<file or folder\>
 - git commit -m \<meaningful commit message\>
 - git push
-- git pull
\ No newline at end of file
+- git pull
+
+## Database connection
+1. Run the npm install
+2. configure the service account (in first iterations we will use a default credetial)
+
+## querys examples
+1. set query (if you don't specify an ID, firebase will create one automatically): 
+db.collection("Cities").doc("ID").set({
+name: "Los Angeles",
+state: "CA",
+country: "USA"
+})
+.then(() => {
+    console.log("Document successfully written!");
+})
+.catch((error) => {
+    console.error("Error writing document: ", error);
+});
+
+2. set query in an existent file:
+var cityRef = db.collection('cities').doc('ID');
+
+var setWithMerge = cityRef.set({
+    capital: true
+}, { merge: true }); 
+
+If you wanna overwrite the existent file just put false in the merge setting.
+
+3. update a query:
+const cityRef = db.collection('cities').doc('ID');
+
+// Set the 'capital' field of the city
+const res = await cityRef.update({capital: true});
+
+Also you can update an object component like this:
+const initialData = {
+  name: 'Frank',
+  age: 12,
+  favorites: {
+    food: 'Pizza',
+    color: 'Blue',
+    subject: 'recess'
+  }
+};
+
+// ...
+const res = await db.collection('users').doc('Frank').update({
+  age: 13,
+  'favorites.color': 'Red'
+});
+
+4. arrayUnion() and arrayRemove() into query:
+const admin = require('firebase-admin');
+// ...
+const washingtonRef = db.collection('cities').doc('DC');
+
+// Atomically add a new region to the "regions" array field.
+const unionRes = await washingtonRef.update({
+  regions: admin.firestore.FieldValue.arrayUnion('greater_virginia')
+});
+// Atomically remove a region from the "regions" array field.
+const removeRes = await washingtonRef.update({
+  regions: admin.firestore.FieldValue.arrayRemove('east_coast')
+});
+
+// To add or remove multiple items, pass multiple arguments to arrayUnion/arrayRemove
+const multipleUnionRes = await washingtonRef.update({
+  regions: admin.firestore.FieldValue.arrayUnion('south_carolina', 'texas')
+  // Alternatively, you can use spread operator in ES6 syntax
+  // const newRegions = ['south_carolina', 'texas']
+  // regions: admin.firestore.FieldValue.arrayUnion(...newRegions)
+});
+
+5. delete a query:
+const res = await db.collection('cities').doc('ID').delete();
+
+6. get a query:
+const cityRef = db.collection('cities').doc('ID');
```

Comment:
I don't think that it's a good idea to put the logic and the way we do the queries on the README file for security reasons

Reply:
that is an example about the database works and everyone can search the same structure.

Reply:
Commonly, README file should contain information about using, setting up, configuring or  understanding the project. Its target audencie is intended to be potential contributors, users or any other people trying to learn about what it does. The `query examples` section here could be more appropriate for a project wiki or database library documentation.

This actually should include just instructions about how the project should be configured for running. Example
```
Setup configuration file:

/config.json

{
    "serviceAccount": <account example>,
    "apiKey": <apiKey>
}

To be provided with an account/key, ask a project admin to provide this information
```